deliveries/Solang_Playground-milestone_4.md

```diff
@@ -0,0 +1,18 @@
+# Milestone Delivery :mailbox:
+
+**The delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/milestone-deliverables-guidelines.md).**
+
+- **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/Solang_Playground.md
+- **Milestone Number:** 4
+
+**Context**
+This milestone involves 
```

Comment:
Were you planning on writing something here, @salaheldinsoliman?

Reply:
@semuelle Ah sorry for that, just updated it